### ISSUE TITLE
Missing Episodes - Additional Filter - Currently Airing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#388]
```
Sometimes when I'm watching alot of stuff that is currently airing it's a bit 
difficult at times to keep track of exactly which episodes I'm missing 
considering with some of the older series I'm missing episodes here and there 
and just haven't gotten to fixing them yet. So yeah being able to filter those 
results a bit more would be helpful.
```

Original issue reported on code.google.com by `imperia...@gmail.com` on 14 Jan 2013 at 2:43

**Comment by GoogleCodeExporter** [comment]
```
Have you tried using the "Finished Airing" filter with "Exclude"?
```

Original comment by `werndly...@gmail.com` on 14 Jan 2013 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
I can do that in collection sure, but in the missing episodes actual utility is 
where I want just a nice simple list that I can read.
```

Original comment by `and...@jeffree.id.au` on 15 Jan 2013 at 1:40

**Comment by GoogleCodeExporter** [comment]
```
ah got it
```

Original comment by `werndly...@gmail.com` on 15 Jan 2013 at 5:59
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.21
```

Original comment by `werndly...@gmail.com` on 29 Jan 2013 at 5:42
- Changed state: **Fixed**
